### ISSUE TITLE
[android] Map stays in App

### ISSUE THREAD

**Issue by vanBrunneren** [react-community/react-native-maps#262]
I have a map in a ScrollView. Sometimes when I scroll over the map, the map don't disappear and a black field stays in the app and can't get removed. The black field is staying in front of the whole app and only disappear if I restart the app. This behavior only appears on android, on iOS everything works perfect.

I made a gif, first I scroll down and the map sticks with a black view, second I hit the back button and go to a different scene in my app:
<iframe src="//giphy.com/embed/xT8qBirsaUH20o8BOg" width="480" height="800" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/xT8qBirsaUH20o8BOg">via GIPHY</a></p>

MapView Component:

```
<MapView
    cacheEnabled={true}
    cacheLoadingBackgroundColor={'#eeeeee'}
    cacheLoadingIndicatorColor={'#666666'}
    style={dealDetailStyle.map}
    region={region}>
    {locations}
</MapView>
```

Style of the MapView:

```
map: {
        position: 'absolute',
        top: 0,
        left: 0,
        right: 0,
        bottom: 0,
        height: 200, 
        width: window.width
    },
```

has anyone an idea what the problem could be in this case?
thanks for any advice

**Comment by allthetime** [comment]
If I had to guess I'd say it's your use of absolute positioning. You should try it with `position: 'relative'` or flex.

**Comment by vanBrunneren** [comment]
this doesn't have any impact. The problem occurs with `position: 'relative'` and with `flex`. The weird thing about this is that the black field stays in the whole app and not only in this component. I really can't explain why this is happening....

**Comment by jrichardlai** [comment]
Not sure what is the issue but using this branch https://github.com/lelandrichardson/react-native-maps/pull/91 you can cache the view so when scrolling it uses an image

**Comment by i8wu** [comment]
@vanBrunneren did you ever find a fix for this?

**Comment by vanBrunneren** [comment]
@i8wu no I changed my app to show the map in a separate component. The bug is still there when I insert a map to a ScrollView. I don't mind if we close this issue because I don't need this feature at the moment, but the bug is still there.

**Comment by haikyuu** [comment]
I experienced the same issue with version 0.8 of `react-native-maps`.
This issue seems to be related with [this one](https://github.com/airbnb/react-native-maps/issues/453). Upgrading to 0.9 or more fixes it.
I think we can close this issue now.

**Comment by cbartondock** [comment]
This is still an issue for me in 12.4

**Comment by alvelig** [comment]
Fixed in #589
